doc/opencv.bib

```diff
@@ -265,7 +365,8 @@ @article{Daniilidis98
   journal = {International Journal of Robotics Research},
   year = {1998},
   volume = {18},
-  pages = {286--298}
+  pages = {286--298},
+  url = {https://typeset.io/pdf/hand-eye-calibration-using-dual-quaternions-1boiebmyns.pdf}
```

Comment:
The link returns 403

Reply:
In my location, the link is OK. I don't know how to proceed.

![image](https://github.com/user-attachments/assets/c06f60cc-22f8-480c-a68d-05a361d14de1)